extensions/amp-story/1.0/amp-story-viewer-messaging-handler.js

```diff
@@ -57,8 +57,8 @@ const GET_STATE_CONFIGURATIONS = {
     property: StateProperty.MUTED_STATE,
   },
   'PAGE_ATTACHMENT_STATE': {
-    dataSource: DataSources.HISTORY,
-    property: HistoryState.ATTACHMENT_PAGE_ID,
+    dataSource: DataSources.STORE_SERVICE,
+    property: StateProperty.ATTACHMENT_PAGE_ID,
```

Comment:
Now that `ATTACHMENT_PAGE_ID` can be null or a number, is there any way to have the `PAGE_ATTACHMENT_STATE` always return a boolean? 

This was previously done by line 144: ` value = !!getHistoryState(this.win_, config.property)`.
It doesn't seem right that the `PAGE_ATTACHMENT_STATE` is a number.

Reply:
+1 to both store and return a boolean so we don't potentially break existing implementations. Good catch :))
If we need the actual ID it's in the history. If Player users want to get the ID they can get the current_page_id.